package.json

```diff
@@ -1,17 +1,22 @@
 {
   "name": "@noblocknoparty/blockparty-contracts",
-  "version": "0.9.0",
+  "version": "1.0.0",
```

Comment:
Do you want to keep `1.0.0` until our VMP launch?

Reply:
What do you mean?